cmd/cspc-operator/app/start.go

```diff
@@ -35,16 +33,18 @@ import (
 )
 
 var (
-	kubeconfig string
+	kubeconfig = flag.String("kubeconfig", "", "Path for kube config")
```

Comment:
how this is used?

Reply:
https://github.com/openebs/maya/pull/1400/files/4b266f439b24bcba9ff6b32a4cddb85129841d07#diff-58f70fa291ffa3b4dabc85117f2761c7R105